dot/types/account.go

```diff
@@ -0,0 +1,19 @@
+package types
+
+import gtypes "github.com/centrifuge/go-substrate-rpc-client/v2/types"
+
+// AccountInfo Information of an account.
+type AccountInfo struct {
+	// The number of transactions this account has sent.
+	Nonce gtypes.U32
```

Comment:
it might be nicer to have this as uint32 so we don't need to import this package every time we want to use this type